packages/boot/tools/supports.ts

```diff
@@ -140,7 +139,7 @@ export const keyArrayEqual = (
  * @param options.configPath - Path to the base config file
  * @param options.defaultManagerType - SwingSet manager type to use
  * @param options.discriminator - Optional string to include in the config filename
- * @returns Path to the generated config file
+ * @returns {string} path to the generated config file
```

Comment:
this is a `.ts` file so the JSDoc return type is ignored.

if an eslint rule is warning about this i think the solution is to revise its configuration

Reply:
OK, makes sense. Done.